src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+	"item.create.rose_quartz_axe": "Złocona kwarcowa siekiera",
+	"item.create.rose_quartz_sword": "Złocone kwarcowe ostrze",
+
+	"block.create.copper_ore": "Ruda miedzi",
+	"block.create.copper_block": "Blok miedzi",
+	"block.create.copper_shingles": "Miedziane gonty",
+	"block.create.zinc_ore": "Ruda cynku",
+	"block.create.zinc_block": "Blok cynku",
+	"block.create.brass_block": "Blok mosiądzu",
+	
+	"block.create.andesite_casing": "Andezytowa skrzynia",
+	"block.create.brass_casing": "Mosiądzowa skrzynia",
+	"block.create.copper_casing": "Miedziana skrzynia",
+
+	"block.create.cogwheel": "Koło zębate",
+	"block.create.large_cogwheel": "Duże koło zębate",
+	"block.create.turntable": "Obrotnica",
+	"block.create.gearbox": "Skrzynia biegów",
+	"block.create.gearshift": "Przekładnia biegów",
+	"block.create.clutch": "Sprzęgło",
+	"block.create.shaft": "Wał",
+	"block.create.encased_belt": "Obudowana taśma",
+	"block.create.encased_shaft": "Obudowany wał",
+	"block.create.encased_fan": "Obudowany wiatrak",
+	"block.create.adjustable_pulley": "Przekładnia cięgnowa",
+	"block.create.nozzle": "Dysza",
+	"block.create.hand_crank": "Korba",
+	"block.create.cuckoo_clock": "Zegar z kukułką",
+	"block.create.creative_motor": "Silnik elektryczny",
+	"block.create.belt": "Taśma mechaniczna",
+	"block.create.millstone": "Koło młyńskie",
+	"block.create.crushing_wheel": "Koło mielnicze",
+	"block.create.drill": "Wiertło mechaniczne",
+	"block.create.portable_storage_interface": "Przenośny interfejs magazynowy",
+	"block.create.harvester": "Podbieracz mechaniczny",
+	"block.create.saw": "Piła mechaniczna",
+	"block.create.water_wheel": "Koło wodne",
+	"block.create.mechanical_press": "Prasa mechaniczna",
+	"block.create.mechanical_mixer": "Mikser mechaniczny",
+	"block.create.deployer": "Ręka mechaniczna",
+	"block.create.basin": "Misa",
+	"block.create.mechanical_crafter": "Rzemielśnik mechaniczny",
+	"block.create.flywheel": "Koło zamachowe",
+	"block.create.furnace_engine": "Silnik spalinowy",
+	"block.create.speed_gauge": "Prędkościomierz",
+	"block.create.stress_gauge": "Stresomierz",
+	"block.create.cart_assembler": "Dźwig wagonowy",
+	"block.create.analog_lever": "Dźwignia analogiczna",
+	"block.create.rotation_speed_controller": "Kontroler prędkości obrotów",
+
+	"block.create.sticky_mechanical_piston": "Lepki tłok mechaniczny",
+	"block.create.mechanical_piston": "Tłok mechaniczny",
+	"block.create.mechanical_piston_head": "Głowa tłoka mechanicznego",
+	"block.create.piston_pole": "Ramie tłoka mechanicznego",
+	"block.create.mechanical_bearing": "Wiertarka mechaniczna",
+	"block.create.clockwork_bearing": "Wiertarka zegarowa",
+	"block.create.rope_pulley": "Krążek linowy",
+	"block.create.rope": "Lina",
+	"block.create.pulley_magnet": "Przekładnia magnesu",
+	"block.create.translation_chassis": "Poziomy korpus",
+	"block.create.rotation_chassis": "Pionowy korpus",
+
+	"block.create.contact": "Wykrywacz pulsu",
```

Comment:
Hmm, could go with _"Kontakt redstone"_ this way we keep the keyword "redstone" which helps in looking things up. It also sticks closer to the translation.

Reply:
"Kontakt" or "Stycznik", as "kontakt" is commonly associated more with a wall socket...

Reply:
Why not "Wykrywacz redstone"?

Reply:
IN Polish translation of Minecraft redstone was translated as "czerwony kamień", now in 1.15 as see its name is kept in most cases as "redstone", so we have "sproszkowany redstone" and "pochodnia redstone".
Repeater is "przekaźnik" and comparator is "komparator", so all those names are very close to english original names.
I would go with "Styk redstone" or "Kontakt redstone" to keep this translation as close to original as possible, same as was done with vanilla redstone elements.